From 871d63c3be529cee2c2bd868750eda121ece46dc Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 29 May 2020 14:43:57 +0200
Subject: [PATCH] Fix CI badges in README.md and fix dracut description

While fixing the CI badges, I removed some old crufty README text.
---
 README.md   | 11 ++++-------
 dracut.spec |  2 +-
 2 files changed, 5 insertions(+), 8 deletions(-)

diff --git a/README.md b/README.md
index 9948bf97..0ab54ea6 100644
--- a/README.md
+++ b/README.md
@@ -4,15 +4,15 @@ dracut
 dracut is an event driven initramfs infrastructure.
 
 [![Build Status](https://travis-ci.org/dracutdevs/dracut.svg?branch=master)](https://travis-ci.org/dracutdevs/dracut)
-[![Fedora-30](https://github.com/dracutdevs/dracut/workflows/Fedora-30/badge.svg?branch=master)](https://github.com/dracutdevs/dracut/actions?query=workflow%3AFedora-30)
 [![Fedora-31](https://github.com/dracutdevs/dracut/workflows/Fedora-31/badge.svg?branch=master)](https://github.com/dracutdevs/dracut/actions?query=workflow%3AFedora-31)
+[![Fedora-32](https://github.com/dracutdevs/dracut/workflows/Fedora-32/badge.svg?branch=master)](https://github.com/dracutdevs/dracut/actions?query=workflow%3AFedora-32)
 
 dracut (the tool) is used to create an initramfs image by copying tools
 and files from an installed system and combining it with the
 dracut framework, usually found in /usr/lib/dracut/modules.d.
 
-Unlike existing initramfs's, this is an attempt at having as little as
-possible hard-coded into the initramfs as possible.  The initramfs has
+Unlike other implementations, dracut hard-codes as little
+as possible into the initramfs. The initramfs has
 (basically) one purpose in life -- getting the rootfs mounted so that
 we can transition to the real rootfs.  This is all driven off of
 device availability.  Therefore, instead of scripts hard-coded to do
@@ -20,10 +20,7 @@ various things, we depend on udev to create device nodes for us and
 then when we have the rootfs's device node, we mount and carry on.
 This helps to keep the time required in the initramfs as little as
 possible so that things like a 5 second boot aren't made impossible as
-a result of the very existence of an initramfs.  It's likely that
-we'll grow some hooks for running arbitrary commands in the flow of
-the script, but it's worth trying to resist the urge as much as we can
-as hooks are guaranteed to be the path to slow-down.
+a result of the very existence of an initramfs.
 
 Most of the initramfs generation functionality in dracut is provided by a bunch
 of generator modules that are sourced by the main dracut script to install
diff --git a/dracut.spec b/dracut.spec
index 1eb47402..4d1cb7a4 100644
--- a/dracut.spec
+++ b/dracut.spec
@@ -99,7 +99,7 @@ Requires: libkcapi-hmaccalc
 
 %description
 dracut contains tools to create bootable initramfses for the Linux
-kernel. Unlike previous implementations, dracut hard-codes as little
+kernel. Unlike other implementations, dracut hard-codes as little
 as possible into the initramfs. dracut contains various modules which
 are driven by the event-based udev. Having root on MD, DM, LVM2, LUKS
 is supported as well as NFS, iSCSI, NBD, FCoE with the dracut-network

